packages/headless/src/app/engine-configuration.ts

```diff
@@ -110,11 +110,15 @@ export interface AnalyticsConfiguration {
    */
   originContext?: string;
   /**
-   * Origin level 2 is a usage analytics event metadata whose value should typically be the name/identifier of the tab from which the usage analytics event originates.
+   * Sets the value of the Origin Level 2 dimension on the analytic events.
    *
-   * When logging a Search usage analytics event, originLevel2 should always be set to the same value as the corresponding tab (parameter) Search API query parameter so Coveo Machine Learning models function properly, and usage analytics reports and dashboards are coherent.
+   * Origin level 2 is a usage analytics event metadata whose value should typically be the name/identifier of the tab from which the usage analytics event originates. 
    *
```

Comment:
To get rid of those linting warnings, make sure your branch is up-to-date with master, then run `npm i `from the root of the project to install the current version of prettier locally, and then `npm run lint:fix`. Commit and push the changes to your branch and you're done (all credits go to @fbeaudoincoveo)